### ISSUE TITLE
Use of network storage paths does not seem to work.

### ISSUE THREAD

**Issue by Dropbear67** [zanllp/sd-webui-infinite-image-browsing#399]
I use the following paths to save images for SD

\\Asustor\ai\DigitalArt\
\\Asustor\ai\DigitalArt\txt2img
\\Asustor\ai\DigitalArt\img2img
\\Asustor\ai\DigitalArt\extras

etc

When I hit "Local" I get
Move failed. Check your path input.C:/ not found

**Comment by zanllp** [comment]
Yes, currently there is an issue with network storage paths in certain cases. I will address this problem in the future.
